FinniversKit/Sources/Recycling/GridViews/NeighborhoodProfile/NeighborhoodProfileBannerView.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+protocol NeighborhoodProfileBannerViewDelegate: AnyObject {
+    func neighborhoodProfileBannerDidSelectButton(_ view: NeighborhoodProfileBannerView)
+}
+
+final class NeighborhoodProfileBannerView: UIView {
+//    static func height(forTitle title: String, width: CGFloat) -> CGFloat {
+//        return title.height(withConstrainedWidth: width, font: titleFont)
+//    }
+//
+//    private static let titleFont = UIFont.title3
+
+    // MARK: - Internal properties
+
+    weak var delegate: NeighborhoodProfileBannerViewDelegate?
+    
+    var text = "" {
+        didSet {
+            textLabel.setHTMLText(text)
+            textLabel.textAlignment = .center
```

Comment:
This is already set when configuring the label.

Reply:
We found out that it's needed, since the label is configured with an `NSAttributedString`, which can have alignment as part of the attributes.

> If you’re using styled text, assigning a new value to this property applies the text alignment to the entirety of the string in the [attributedText](https://developer.apple.com/documentation/uikit/uilabel/1620542-attributedtext) property.